src/pages/YC_Liew/Dota2.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import "./Dota2.scss"
+
+function Dota2() {
+
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
+    fetchItems();
+  }, []);
+
+  const fetchItems = async () => {
+    const data = await fetch(
+      'https://api.opendota.com/api/constants/items'
+    );
+    const object = await data.json();
```

Comment:
you guy should consider write a common function to fetch api and handle response & error